src/nxt_http_parse.c

```diff
@@ -1223,6 +1224,7 @@ nxt_http_fields_hash_collisions(nxt_lvlhsh_t *hash,
 
     lhq.replace = 0;
     lhq.proto = &proto;
+    lhq.pool = NULL;
 
     mask = level ? (1 << NXT_HTTP_FIELD_LVLHSH_SHIFT) - 1 : 0xFFFF;
 
```

Comment:
The above applies here also...

```c
    proto = nxt_http_fields_hash_proto;
    ...
    lhq.proto = &proto;
```

So lhq.proto.alloc = nxt_lvlhsh_alloc which doesn't use lhq.pool...

Reply:
This one is not important the first one is important.